src/Navbar.js

```diff
@@ -11,10 +11,13 @@ function handleClick(){
 }
     return (  
         <div className="navbar">
+            <img src="webl.png" alt="React Logo" className="logo"/>
```

Comment:
This image should link back to the home directory or the '/' directory.

Reply:
I have done this sir